packages/markdownviewer/style/index.css

```diff
@@ -1,35 +0,0 @@
-/*-----------------------------------------------------------------------------
-| Copyright (c) Jupyter Development Team.
-| Distributed under the terms of the Modified BSD License.
-|----------------------------------------------------------------------------*/
-
-:root {
-  --jp-private-markdownviewer-padding: 16px;
-}
-
-.jp-MarkdownViewer {
-  border-top: var(--jp-border-width) solid var(--jp-border-color2);
-  outline: none;
```

Comment:
We will need to make sure the overflow is working properly so that it wraps/scrolls as expected.